### ISSUE TITLE
VoiceOver not annoucening aria-valuenow for spin button example

### ISSUE THREAD

**Issue by jongund** [w3c/aria-practices#1187]
@mcking65 
@cookiecrook 

For VoiceOver on macOS for both Safari and Chrome, VO is not announcing the value of spinbuttons using only `aria-valuenow` (e.g. the "Year" spin button), it only seems to work when `aria-valuetext` is also present (e.g. "Day" and "Month" spinbuttons).

[Date Picker Spin Button Example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/spinbutton/datepicker-spinbuttons.html)

I did some additional testing by removing the aria-valuetext property from the "day" and "month" spin buttons and VO then did not announce the values for "day" or "month" either.  When I added `aria-valuetext` to the "Year" spinbutton value is read by VO.

**Comment by mcking65** [comment]
@jongund,

We are not collecting screen reader bugs in the APG issues. As far as I know, there is not a public repository for VoiceOver bugs. The only way I know how to report them is to send email to accessibility@apple. Is that correct @cookiecrook?

**Comment by ZoeBijl** [comment]
For completion sake:

You can also report bugs via [Apple’s feedback form](https://www.apple.com/feedback/macos.html) or via [Apple’s developer bug reporter](https://feedbackassistant.apple.com/welcome). You don’t need an account for the feedback form but will need one for the bug reporter.

**Comment by aleventhal** [comment]
I filed a bug on Chrome: https://bugs.chromium.org/p/chromium/issues/detail?id=1010500
This sounds like something we can fix on the Chrome side of things.

**Comment by jongund** [comment]
I filed a bug with the Apple feedback form:
https://www.apple.com/feedback/macos.html
